.drone.yml

```diff
@@ -227,20 +227,6 @@ steps:
           - feature/*
       event: pull_request
 
-  - name: anchore_scan
-    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/anchore-submission:latest
-    pull: always
-    environment:
-      IMAGE_NAME: paf:${DRONE_COMMIT_SHA}
-      LOCAL_IMAGE: true
-      TOLERATE: medium
-      WHITELIST_FILE: hof-services-config/Public_Allegation_Form/anchore-cve-exceptions.txt
-    when:
-      branch:
-        include:
-          - master
-          - feature/*
-      event: pull_request
```

Comment:
Can we also get rid of the 'cron-anchore-scan' step and 'anchore-submission-server' service at the same time? (further down to the bottom of the file)

Reply:
Sounds good.